src/storybook/stand-alone-documentaion/motion/motion.stories.mdx

```diff
@@ -1,13 +1,148 @@
 import { Meta } from "@storybook/addon-docs";
 import { ComponentName } from "../../components";
+import { AnimationOverviews } from "./animation-overviews/animation-overviews";
+import { DurationProductiveTokens } from "./duration-productive-tokens/duration-productive-tokens";
+import { DurationExpressiveTokens } from "./duration-expressive-tokens/duration-expressive-tokens";
+import { DurationTokenTable } from "./duration-token-table/duration-token-table";
+import { EasingOverviews } from "./easing-overviews/easing-overviews";
+import { EasingTokenTable } from "./easing-token-table/easing-token-table";
+import { AnimationTypes } from "./animation-types/animation-types";
+import {
+  EasingDo,
+  EasingDont,
+  ConsistencyDo,
+  ConsistencyDont,
+  SimplicityDo,
+  SimplicityDont,
+  DontDelayDo,
+  DontDelayDont
+} from "./assets";
 import { ComingSoon } from "../../components/coming-soon/coming-soon";
 
-<Meta
-  title="Foundations/Motion"
+<Meta title="Foundations/Motion" />
+
+<ComponentName withFoundationBackground>Motion</ComponentName>
+
+- [Overview](#overview)
+- [Implementation](#implementation---duration-and-ease)
+- [Do's and dont's](do's-and-don'ts)
+
+## Overview
+
+Animation is a powerful tool for communication and is an integral part of the visual language of a product. It helps to make the user interface easier to understand, navigate, intuitive to use and more interactive to work with. Here are the 4 main purposes that helps bridging between the virtual world we build and the physical world we know:
```

Comment:
copy/grammer changes which suggest by grammarly:

Animation is a powerful tool for communication and is an integral part of the visual language of a product. It helps make a user interface easier to understand, navigate, intuitive to use, and more interactive to work with. Here are the four main concepts that help bridge between the virtual world we build and the physical world we know:
Orientation & Continuity
Motion gives us a familiar feeling of an environment by helping us understand where things are located, where they're coming from, and where to find them again.